spec/factories/bills.rb

```diff
@@ -0,0 +1,27 @@
+FactoryBot.define do
+  factory :bill do
+    amount { Faker::Number.decimal(l_digits: 2) }
+    date { Faker::Time.between(from: DateTime.now - 1, to: DateTime.now) }
```

Comment:
what means `-1`, i think this would not work, better to use `1.day, 1.second`, etc..

Reply:
or `between(from: 1.day.ago, to: Datetime.now)`